lib/vets/shared_logging.rb

```diff
@@ -28,20 +28,62 @@ def log_exception_to_sentry(exception, extra_context = {}, tags_context = {}, le
     def log_message_to_rails(message, level, extra_context = {})
       # this can be a drop-in replacement for now, but maybe suggest teams
       # handle extra context on their own and move to a direct Rails.logger call?
-      formatted_message = extra_context.empty? ? message : "#{message} : #{extra_context}"
-      Rails.logger.send(level, formatted_message)
+      #
+      # Code was previously using Rails.logger.send(level, formatted_message) which could cause an exception
+      # of its own. Hacky, but safe. Rails.logger.info etc do the right thing.
+
+      # if level is passed as a symbol (e.g. :warn), handle it. convert to string
+
+      level = level.to_s.downcase
+      level = 'warn' if level == 'warning' # Rails doesn't support Sentries Warning level
```

Comment:
there's a couple places where normalizing the level happens, once here and later on either side of the `normalize_shared_level` call in `log_exception_to_rails` in ruby you can pass an optional code block to a method and then call it via `yield`, the first line creates a lambda for resuse:
```ruby
def normalize_level_for_rails(level)
  normalize = ->(lvl) { lvl.to_s.downcase == 'warning' ? 'warn' : lvl.to_s.downcase }
  return normalize.call(level) unless block_given?
  processed_level = yield(normalize.call(level))
  normalize.call(processed_level)
end
```
Here you would call it without the block:
```ruby
level = normalize_level_for_rails(level)
```

Reply:
`normalize_shared_level` is a DANGEROUS method and someone wrote it with many holes. For example, it may return `blahblah`. I didn't want to touch it because `log_message_to_sentry` uses it. Didn't want to break that piece of code